python/graph_coloring.py

```diff
@@ -1,19 +1,23 @@
 import mgp
 from collections import defaultdict
-from typing import List, Optional
+from typing import List, Optional, Dict, Any
+import mage.graph_coloring_module
+from mage.graph_coloring_module import Parameter
 from mage.graph_coloring_module import Graph
-from mage.graph_coloring_module import QA
-from mage.graph_coloring_module import ConflictError
-from mage.graph_coloring_module import SimpleMutation
-from mage.graph_coloring_module import MultipleMutation
-from mage.graph_coloring_module import LDO
-from mage.graph_coloring_module import SDO
+
+
+def _str2Class(name):
+    return getattr(mage.graph_coloring_module, name)
 
 
 @mgp.read_proc
 def color_graph(
     context: mgp.ProcCtx, parameters: mgp.Map
 ) -> mgp.Record(node=str, color=str):
+    """
+    Example:
+    CALL graph_coloring.color_graph({}) YIELD *;
```

Comment:
If you don't want your procedure call to look like this, simply put the default parameter in the function call
```parameters: mgp.Map = {}```
This would look much better when calling with default parameters:
```CALL graph_coloring.color_graph() YIELD *;```